website/blog/2023-01-02-aluminium.md

```diff
@@ -0,0 +1,922 @@
+  needed prefix inline. [dos65](https://github.com/dos65)
+- bugfix: Fix detecting companions for classes defined directly in method for
+  document highlight and rename.
+- bugfix: exhaustive match when using renamed symbol.
+  [jkciesluk](https://github.com/jkciesluk)
+- bugfix: Retry starting Ammonite when the script file changed and was fixed.
+- bugfix: Properly highlight package object name.
+- bugfix: Properly highlight named arguments in case class constructors.
+  [jkciesluk](https://github.com/jkciesluk)
+- fix: automatically import companion object instead of apply method.
+  [jkciesluk](https://github.com/jkciesluk)
+- feature: Add showing semanticdb for sbt files and worksheets.
+  [jkciesluk](https://github.com/jkciesluk)
+- bugfix: Correctly display statuses for Ammonite scripts in Doctor.
+- bugfix: Don't show string code actions when they are enclosed within selected
+  range.
+- bugfix: [Scala 3] Allow to automatically complete already overridden symbols.
+  [jkciesluk](https://github.com/jkciesluk)
+- bugfix: [Scala 3] Make implementAll work properly with type aliases and
+  higher. kind type[jkciesluk](https://github.com/jkciesluk)
+- bugfix: [Scala 3] use braces for match completion with -no-indent option.
+  [jkciesluk](https://github.com/jkciesluk)
+- bugfix: Take into account the length of interpolator for code actions related.
+  to string interpolation.
+- bugfix: Make multistage Mill scripts work.
+- bugfix: implement methods with path-dependent type arguments.
+- bugfix: Properly reindex build after reloading sbt.
+- bugfix: Recommend newer RC Scala versions when project is using a RC
+  [rkrzewski](https://github.com/rkrzewski)
+- chore: Try and print all dependencies used by Metals.
+- bugfix: [Scala 3] Fix highlight for extension methods.
+  [jkciesluk](https://github.com/jkciesluk)
+- improvement: Improve behaviour of Initial ivy Completions when backticks are
+  not added. [LaurenceWarne](https://github.com/LaurenceWarne)
+- bugfix: Call hierarchy incoming calls includes also incoming calls of super.
+  methods [kasiaMarek](https://github.com/kasiaMarek)
+- feature: Show @main lenses with scala cli scripts.
+  [LaurenceWarne](https://github.com/LaurenceWarne)
+- bugfix: Don't encode paths on Windows to fix issues with dependencies in
+  directories with non alphanumeric characters.
+- bugfix: Fix document highlighting mechanism for Java files.
+  [MaciejG604](https://github.com/MaciejG604)
+- improvement: Better folding of definitions and overall improvements to folded
+  regions. [MaciejG604](https://github.com/MaciejG604)
+- bugfix: stop watching generated files to avoid compilation loop.
+  [adpi2](https://github.com/adpi2)
+
+## Contributors
+
+Big thanks to everybody who contributed to this release or reported an issue!
+
+```
+$ git shortlog -sn --no-merges v0.11.9..v0.11.10
+    65	Tomasz Godzik
+    40	Jakub Ciesluk
+    47	scalameta-bot
```

Comment:
change order?
```suggestion
    47	scalameta-bot
    40	Jakub Ciesluk
```

Reply:
Sure, though I am on the fence whether to even mention the bots :thinking: